### ISSUE TITLE
embedded comment parse fail

### ISSUE THREAD

**Issue by Pramy** [emicklei/proto#131]
test.proto
```protobuf
syntax = "proto3";
package test;
import "google/protobuf/descriptor.proto";

message FooOption {
  optional bool from = 1;
  optional bool to = 2;
}
extend google.protobuf.FieldOptions {

  optional FooOption foo_option = 1;
}

message Foo {

  message  /* ddd

  */
      InnerFoo {
    optional  // aaa
        uint64 /* bbb
     */value = 1 // ccc
    [(foo_option) // 123
        = {from: true, // kkkkk
      to: true}
      // 123123
    ];
  }
}
```
main.go
```go
package main

import (
	"fmt"
	"github.com/emicklei/proto"
	"os"
)

func main() {
	reader, _ := os.Open("test.proto")
	defer reader.Close()

	parser := proto.NewParser(reader)
	definition, err := parser.Parse()
	if err != nil {
		fmt.Println(err)
	}

	proto.Walk(definition,
		proto.WithService(handleService),
		proto.WithMessage(handleMessage))
}

func handleService(s *proto.Service) {
	fmt.Println(s.Name)
}

func handleMessage(m *proto.Message) {
	fmt.Println(m.Name)
}

```
output
```
<input>:16:12: found "/* ddd\n\n  */" but expected [message identifier]
FooOption
google.protobuf.FieldOptions
```

**Comment by ernest-ag5** [comment]
thank for reporting this issue.  It got from my attention radar. Will have a look at it soon

**Comment by emicklei** [comment]
i am closing this as the issue addresses a very uncommon case. Handling this would complicate the package even more.

**Comment by zack-littke-smith** [comment]
I ran into this as well, and I am sad to see this resolution. This parser should handle proto files as intended, not disregarding edge cases as being unlikely

**Comment by emicklei** [comment]
> @emicklei
> 
> I ran into this as well, and I am sad to see this resolution. This parser should handle proto files as intended, not disregarding edge cases as being unlikely.

At the time of reporting this issue, I found that it was a usecase that very synthesized, almost with the purpose of finding a problem with the parser (like fuzzing). Next it wasn't given any attention for over a year, so I did not felt it was urgent enough to spent much time on it. 

So, maybe if you @zack-littke-smith can provide me with an example to replicates your problem, we could work on a solution for that first.

**Comment by zack-littke-smith** [comment]
I discovered this via usage of this parser in a downstream library - it is not yet clear to me if the fault is with this parser or that library. Copying my example from [the issue](https://github.com/nilslice/protolock/issues/154#issue-2073695560) I filed with them:
```
message Example {
  optional /* i'm a comment */ bool field = 1;
}
```
Seems to not be parsed properly, and seems like a not unreasonable place for a comment. Though again, I am not sure if this parsing library is the one at fault here.
